model/status_diff.go

```diff
@@ -108,17 +109,24 @@ func StatusDiffBuilds(original, patch *build.Build) (BuildStatusDiff, error) {
 		}
 		diff.Tasks = append(diff.Tasks, newDiff)
 	}
+
 	return diff, nil
 }
 
 // getTestUrl returns the correct relative URL to a test log, given a
 // TestResult structure
 func getTestUrl(tr *task.TestResult) string {
-	// Return url if it exists. If there is no test, return empty string.
-	if tr.URL != "" || tr.LogId == "" { // If LogId is empty, URL must also be empty
+	if tr.URL != "" {
 		return tr.URL
 	}
-	return TestLogPath + tr.LogId
+	if tr.LogId != "" {
```

Comment:
I don't really understand this change. The original comment says if LogId is empty, URL must be empty. Why does this need to be changed?

Reply:
With cedar test logs log id will not exist and we use `task_id/execution/test_name` in the `test_log` query.

https://github.com/evergreen-ci/evergreen/blob/main/service/ui.go#L325